src/pages/Welcome/welcomeStyles.css

```diff
@@ -0,0 +1,21 @@
+.welcomeScreen {
+  background: linear-gradient(
+      180deg,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(255, 255, 255, 0.17) 28.26%,
+      rgba(255, 255, 255, 0) 100%
+    ),
+    #29620f;
+  position: absolute;
```

Comment:
I think the positioning might be the reason why .welcomeScreen does not scale to fit the content on smaller screens: 
![image](https://user-images.githubusercontent.com/68360696/120076801-6b5e9780-c075-11eb-8e6f-b0fa7fff5a3f.png)

The white border you have without positioning is there because .App has padding of 3.5em. If you set 
```
margin: -3.5em;
width: 100%; 
```
on .welcomeScreen, it will fill in the entire container and scale together with it, so you won't need to position it.

Reply:
Having the margin on App seems to be causing mobile responsive issues. I know @cafloyd mentioned @kkatoch89 was tackling the responsive issues. But if you removed the margin from `.App` and added padding to the parent container of App. You get the same look without having that overflow issue. And you don't need to worry about negative margins on certain screensizes.

This addresses the comment above as well: https://github.com/the-collab-lab/tcl-22-smart-shopping-list/pull/67#pullrequestreview-671710160. 

<details><summary>Before adding padding to parent container </summary>

<img width="1306" alt="Screen Shot 2021-05-29 at 9 33 55 am" src="https://user-images.githubusercontent.com/3861088/120077886-88896b00-c061-11eb-9cc5-e11e9671ca7d.png">
<img width="1392" alt="Screen Shot 2021-05-29 at 9 37 14 am" src="https://user-images.githubusercontent.com/3861088/120077884-86271100-c061-11eb-96e6-b7221c2c8670.png">

</details>

<details><summary>After adding padding to parent container</summary>
<img width="1350" alt="Screen Shot 2021-05-29 at 9 34 08 am" src="https://user-images.githubusercontent.com/3861088/120077821-1e70c600-c061-11eb-91d2-d21724cfd6bd.png">

<img width="1072" alt="Screen Shot 2021-05-29 at 9 34 11 am" src="https://user-images.githubusercontent.com/3861088/120077818-1c0e6c00-c061-11eb-8bef-f90b6b1d50f7.png">
</details>

Reply:
great tip, @dariatsvetkova ! adding that code got me 3/4 of the way there, but I'm wondering if it's better to figure out how to pull the bottom all the way down, or to add responsive styling to the Welcome screen to address the overflow in mobile. 
![image](https://user-images.githubusercontent.com/58532369/120077893-b4fdb100-c07a-11eb-933b-48c2846f9e1e.png)

Reply:
Oh I missed your comment somehow, @jonathanyeong !! Thank you thank you, that is super helpful!!

Reply:
Thanks @dariatsvetkova and @jonathanyeong !! A combination of both of your feedback did the trick! Now I just to fix the contents to be more responsive, unless we like this style decision 😅
![image](https://user-images.githubusercontent.com/58532369/120078954-d4e3a380-c07f-11eb-88ff-deb95f5183c6.png)